EasyPost/Tracker.cs

```diff
@@ -73,5 +73,19 @@ public class Tracker : Resource {
 
             return request.Execute<Tracker>();
         }
+
+        /// <summary>
+        /// Create a list of trackers
```

Comment:
Per the new linting, this should have a tab
```suggestion
        ///     Create a list of trackers
```